packaging/installer/netdata-updater.sh

```diff
@@ -102,6 +102,8 @@ update() {
 	fi
 
 	info "Re-installing netdata..."
+	REINSTALL_COMMAND="${REINSTALL_COMMAND//--auto-update/}"
+	REINSTALL_COMMAND="${REINSTALL_COMMAND//-u/} -u"
```

Comment:
This will remove `-u` from any command line option that starts with `-u` or contains `-u`.
A cleaner approach is to at least include a space after `//`.

The best approach would be to always append `-u` without removing anything and modify the installer to append options to `REINSTALL_COMMAND` at the point they are parsed, so that it will always append the long names of the options with a space before and after them, and make sure that all options appear just once, no matter how many times they are given at the command line.